lua/ibl/init.lua

```diff
@@ -187,10 +187,15 @@ M.refresh = function(bufnr)
     end
 
     local scope
+    local scope_start_line, scope_end_line
     if not scope_disabled and config.scope.enabled then
         scope = scp.get(bufnr, config)
         if scope and scope:start() >= 0 then
-            offset = top_offset - math.min(top_offset - math.min(offset, scope:start()), config.viewport_buffer.max)
+            local scope_start = scope:start()
+            local scope_end = scope:end_()
+            offset = top_offset - math.min(top_offset - math.min(offset, scope_start), config.viewport_buffer.max)
+            scope_start_line = vim.api.nvim_buf_get_lines(bufnr, scope_start, scope_start + 1, false)
+            scope_end_line = vim.api.nvim_buf_get_lines(bufnr, scope_end, scope_end + 1, false)
```

Comment:
```suggestion
            scope_start_line = vim.api.nvim_buf_get_lines(bufnr, scope_start, scope_start + 1, false)[1]
            scope_end_line = vim.api.nvim_buf_get_lines(bufnr, scope_end, scope_end + 1, false)[1]
```

It's cleaner to do this here.

Reply:
I fixed this.